manim/scene/scene.py

```diff
@@ -849,6 +849,8 @@ def play_internal(self, skip_rendering=False):
         for animation in self.animations:
             animation.finish()
             animation.clean_up_from_scene(self)
+        if not self.renderer.skip_animations:
+            self.update_mobjects(0)
```

Comment:
**Question** : why 0? why updating at the last t?

Reply:
I don't really know much details (because I found this by going back into commits history), but after an animation is finished we need to update the mobjects to avoid this bug. To update the mobjects now, update_mobjects(0) calls mobjects.update(0), which will set dt=0. (btw default value for dt is 0 if nothing is mentioned when calling mobjects.update())

Reply:
I see your point, and I don't think it does have any importance but I think that it'd make more sense to update with the last t, no? 
Did you try?

Reply:
I thought the same but didn't try coz of lack of time, got exams this week

Reply:
No problem! 

good luck !

Reply:
So, did you try it?

Reply:
isn't the mobjects.update(0) already only running when an animation ends?